### ISSUE TITLE
Parse cmetrics from Fluent Bit v1.9

### ISSUE THREAD

**Issue by kubotat** [calyptia/fluent-plugin-cmetrics#5]
# Description of the issue
I've tested Fluent Bit v1.9 and found issues when parsing metrics from Fluent Bit v1.9 with cmetrics parser. It seems there are changes in cmetrics format like the following :
- node_exporter_metrics output of Fluent Bit v1.8.12
```
# HELP node_cpu_seconds_total Seconds the CPUs spent in each mode.
# TYPE node_cpu_seconds_total counter
node_cpu_seconds_total{cpu="0",mode="idle"} 15550319.85 1648704606281
node_cpu_seconds_total{cpu="0",mode="iowait"} 414.05000000000001 1648704606281
node_cpu_seconds_total{cpu="0",mode="irq"} 188411.42999999999 1648704606281
node_cpu_seconds_total{cpu="0",mode="nice"} 2494.3499999999999 1648704606281
node_cpu_seconds_total{cpu="0",mode="softirq"} 4968.3400000000001 1648704606281
node_cpu_seconds_total{cpu="0",mode="steal"} 46432.690000000002 1648704606281
node_cpu_seconds_total{cpu="0",mode="system"} 314173.71000000002 1648704606281
node_cpu_seconds_total{cpu="0",mode="user"} 166170.23000000001 1648704606281
node_cpu_seconds_total{cpu="1",mode="idle"} 15593238.82 1648704606281
node_cpu_seconds_total{cpu="1",mode="iowait"} 1823.0799999999999 1648704606281
node_cpu_seconds_total{cpu="1",mode="irq"} 175412.66 1648704606281
node_cpu_seconds_total{cpu="1",mode="nice"} 2580.3299999999999 1648704606281
node_cpu_seconds_total{cpu="1",mode="softirq"} 4583.8699999999999 1648704606281
node_cpu_seconds_total{cpu="1",mode="steal"} 45426.239999999998 1648704606281
node_cpu_seconds_total{cpu="1",mode="system"} 300571.13 1648704606281
node_cpu_seconds_total{cpu="1",mode="user"} 157451.53 1648704606281
```
- node_exporter_metrics output of Fluent Bit v1.9.0
```
# HELP node_cpu_seconds_total Seconds the CPUs spent in each mode.
# TYPE node_cpu_seconds_total counter
node_cpu_seconds_total{cpu="0",mode="idle"} 15550291.52
node_cpu_seconds_total{cpu="0",mode="iowait"} 414.05000000000001
node_cpu_seconds_total{cpu="0",mode="irq"} 188410.94
node_cpu_seconds_total{cpu="0",mode="nice"} 2494.3499999999999
node_cpu_seconds_total{cpu="0",mode="softirq"} 4968.3199999999997
node_cpu_seconds_total{cpu="0",mode="steal"} 46432.550000000003
node_cpu_seconds_total{cpu="0",mode="system"} 314172.83000000002
node_cpu_seconds_total{cpu="0",mode="user"} 166169.84
node_cpu_seconds_total{cpu="1",mode="idle"} 15593210.630000001
node_cpu_seconds_total{cpu="1",mode="iowait"} 1823.0799999999999
node_cpu_seconds_total{cpu="1",mode="irq"} 175412.10000000001
node_cpu_seconds_total{cpu="1",mode="nice"} 2580.3299999999999
node_cpu_seconds_total{cpu="1",mode="softirq"} 4583.8599999999997
node_cpu_seconds_total{cpu="1",mode="steal"} 45426.139999999999
node_cpu_seconds_total{cpu="1",mode="system"} 300570.19
node_cpu_seconds_total{cpu="1",mode="user"} 157450.98000000001
```
When I put node_exporter_metrics from Fluent Bit v1.9, I found the following error messages:
```
2022-03-31 05:40:10 +0000 [info]: Worker 0 finished unexpectedly with signal SIGABRT
```
# How to reproduce
- Fluent Bit configuration:
```
[SERVICE]
    Flush      5
    Daemon     off
    Log_Level  info

[INPUT]
    Name fluentbit_metrics
    Tag fluentbit.metrics
    scrape_on_start true
    scrape_interval 30

[OUTPUT]
    Name          forward
    Match         *
    Host          <fluentd destination>
    Port          24224
```
- Fluentd configuration:
```
<source>
  @type forward
  port 24224
  bind 0.0.0.0
</source>

<filter fluentbit.metrics>
  @type cmetrics_parser
  format_to_splunk_metric true
  dimensions_key dims
  <fields>
    hostname
    aggregator
  </fields>
</filter>

<match fluentbit.metrics>
  @type stdout
</match>
```

**Comment by cosmo0920** [comment]
I also reproduced that the following configurations:

```ini
[SERVICE]
    Flush      5
    Daemon     off
    Log_Level  info

[INPUT]
    Name node_exporter_metrics
    Tag fluentbit.metrics
    scrape_interval 30

[OUTPUT]
    Name          forward
    Match         *
    Host          <fluentd destination>
    Port          24224
```

```aconf
<source>
  @type forward
  port 24224
  bind 0.0.0.0
</source>

<filter fluentbit.metrics>
  @type cmetrics_parser
  format_to_splunk_metric true
  dimensions_key dims
  <fields>
    hostname
    aggregator
  </fields>
</filter>

<match fluentbit.metrics>
  @type stdout
</match>
```

**Comment by cosmo0920** [comment]
This could be fixed by [cmetrics-ruby v0.3.0.rc](https://rubygems.org/gems/cmetrics/versions/0.3.0.rc)

**Comment by cosmo0920** [comment]
Now fixed at master.
